added arithmetic expression testcases

thepowersss opened the pull request:
attempt 1 (actually attempt 2 but we don't talk about attempt 1)

thepowersss committed:
added arithmetic expression testcases

attempt 1 (actually attempt 2 but we don't talk about attempt 1)

thepowersss commented:
this is sherwin by the way

justinnhli commented:
* `comment interruption` is not quite right. The problem is that the comment doesn't stop until the newline, so it's "eating" everything else
* `newline everywhere` is missing the final semicolon.
* `pemdas test 1` has an output file that doesn't match the comment ;)
* `print nothing` is a syntax error - you are *required* to have an expression in the print statement.

These are clever test cases though - good work! Fix them and tag me again.

thepowersss committed:
arithmetic expression testcases edit 1

first edit

thepowersss committed:
arithmetic expression testcases edit 2

edit 2, changed name of print_nothing to print_zero

thepowersss committed:
arithmetic expression testcases edit 3

edit 3: amended big_number_test to be less than 2 billion to be a valid java int, added negative number tests

justinnhli commented:
One small error: your `quintuple_negatives` case is missing a close paren. Fix that, and I'll merge it in!
